### PR TITLE
ui: making label color match tooltip color for better visibility

### PR DESCRIPTION
**Pull request by dianasaur323**
Fixes https://github.com/cockroachdb/cockroach/issues/16821

### CODE REVIEW

File: pkg/ui/styl/shame.styl [hunk 1]
```diff
@@ -111,7 +111,7 @@
 
   .nv-axislabel
     font-family Lato-Regular
-    fill $button-border-color
+    fill tooltip-color
```
Comment (vilterp) [inline pkg/ui/styl/shame.styl]: is this actually resolving as a variable? variables have to start w/ dollar signs AFAIK

**Comment by vilterp** [comment]
Cool. Screenshot plz?

**Comment by dianasaur323** [comment]
image

File: pkg/ui/styl/shame.styl (hunk 1 above)
Comment (dianasaur323) [reply pkg/ui/styl/shame.styl]: fixed!

**Review by vilterp — APPROVED** [review]
Nice. Squash into one commit before merging (`git rebase -i master`; change second commit from `pick` to `f`)

**Comment by dianasaur323** [comment]
whoa, that's pretty cool. I finally understand what that means now. Merging when green!

**Pull request merged**